examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -79,42 +83,17 @@ export class TaskList extends DataObject implements ITaskList {
         // To add a task, we update the root SharedDirectory.  This way the change is propagated to all collaborators
         // and persisted.  In turn, this will trigger the "valueChanged" event and handleTaskAdded which will update
         // the this.tasks collection.
+        this.root.set(id, { id, name: nameString.handle, priority: priorityCell.handle });
```

Comment:
With this PR, this.root should not store this data anymore (it should only be stored in the two maps below).